stylesheets/components/image.css

```diff
@@ -0,0 +1,15 @@
+.img {
+  height: 150px;
+  width: 150px;
+  object-fit: cover;
+}
+
+.img-responsive {
+  width: 100%;
+  height: auto;
+  object-fit: cover;

```

Comment:
Avoid using the same code multiple times.

Reply:
`img` class is not inherited in the responsive image class. That's why need to write both the place.